tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+Force Tags       Sanity
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          JupyterLibrary
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Variables ***
+@{notebook_images}  s2i-minimal-notebook  s2i-generic-data-science-notebook  tensorflow  pytorch
+
+*** Test Cases ***
+Verify Installed Libraries
+  [Tags]  Sanity
+  ...     ODS-340  ODS-695  ODS-204  ODS-205  ODS-206  ODS-207  ODS-215  ODS-216  ODS-217  ODS-218
+  Wait for RHODS Dashboard to Load
+  ${version-check} =  Is RHODS Version Greater Or Equal Than  1.4.0
```

Comment:
it might be worth to have a common keyword containing lines from 19 to 24

Reply:
Same as above, it could be done but I'd like to avoid changing more than what needs to be changed with this PR. Further changes can be applied during refactoring.